packages/theme/components/Content/FeaturedArticle.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <SfHeading :title="section.title"/>
+    <div v-html="section.rteContent"
+    />
+    <SfButton
+      @click="router.push(section.link[0])"
```

Comment:
We should use link property of SfButton - this will result in `<a href="xx">` tag which is important for SEO

Reply:
Besides that I think it would be good to wrap this path in `localePath()` (and other paths too)